src/components/cms/SkillFeedbackCard/SkillFeedbackCard.js

```diff
@@ -153,85 +153,94 @@ class SkillFeedbackCard extends Component {
     let userFeedbackCard = null;
     let feedbackCards = null;
     if (skillFeedbacks) {
-      feedbackCards = skillFeedbacks.map((data, index) => {
-        userEmail = data.email;
-        userAvatarLink = data.avatar;
-        userName = data.userName;
-        const name = userName === '' ? userEmail : userName;
-        const avatarProps = {
-          src: userAvatarLink,
-          name,
-        };
-        if (accessToken && email && userEmail === email) {
-          userFeedbackCard = (
-            <div key={index}>
+      feedbackCards = []
+        .concat(skillFeedbacks)
```

Comment:
Won't it work without concat function?
I mean skillFeedbacks.sort().map()

Reply:
If used without it there are chances of mutating the original state

Reply:
okay 👍